src/servers/sapphire_zone/Zone/InstanceContent.cpp

```diff
@@ -238,8 +240,24 @@ void Core::InstanceContent::updateInstanceObj( Core::Entity::InstanceObjectPtr o
    if( !object )
       return;
 
-   for( const auto& playerId : m_playerMap )
+   for( const auto& playerIt : m_playerMap )
    {
       // send that packet with le data
+      ZoneChannelPacket< FFXIVIpcObjectControl > eobjStatePacket( playerIt.second->getId() );
+      eobjStatePacket.data().objKind = object->getObjKind();
+      eobjStatePacket.data().state = object->getState();
+      eobjStatePacket.data().objId = object->getId();
+      eobjStatePacket.data().hierachyId = object->getHierachyId();
+      eobjStatePacket.data().position = object->getPos();
+
+      // ????
+      //eobjStatePacket.data().levelId = 4236873;
+      //eobjStatePacket.data().unknown2 = 5;
+      //eobjStatePacket.data().unknown1C = 1065353216;
+      //eobjStatePacket.data().unknown20 = 2147423605;
+      //eobjStatePacket.data().actorId = 1074105831;
+      //eobjStatePacket.data().unknown = 1;
+
+      playerIt.second->queuePacket( eobjStatePacket );
```

Comment:
If we just add another object container to the zone and override the "spawn" in InstanceObject" we could just use existing in range logic for spawning and updating objects.

Reply:
yeah i'm just wondering whats the best way to store objects in Zone properly so we can have multiple copies of things in the event of gathering points and etc

multiple players on the same gathering point and etc, not sure about how we would handle this with the current implementation in InstanceContent

Reply:
gathering points are a whole other can of worms. They have a different type too, so they do not even qualify as eventObject.